src/textarea/Textarea.tsx

```diff
@@ -237,6 +247,7 @@ export default class Textarea extends PureComponent<TextAreaProps, TextAreaState
       <div className={cc(['kirk-textarea', prefix({ error: !!error, disabled }), className])}>
         {label && <label htmlFor={id}>{label}</label>}
         <div
+          onClick={this.onWrapperClick}
```

Comment:
Just wondering: is there also a way to pass a `onClick` handler to the component? If not, do we want to add that for future need?

Reply:
Let's wait to actually need that before adding it.